source/client-side-encryption/tests/README.rst

```diff
@@ -2693,302 +2694,343 @@ Create a ClientEncryption object named ``clientEncryption`` with these options:
+Use ``encryptedClient`` to run a "find" operation on the ``db.explicit_encryption`` collection with the filter ``{ "$expr": <findPayload> }`` and sort the results by ``_id``.
+
+Assert the following two documents are returned:
+
+.. code:: javascript
+
+   // Convert 0 and 6 to the encrypted field type
+   { "_id": 0, "encrypted<Type>": 0 }
+   { "_id": 1, "encrypted<Type>": 6 }
 
-Assert that these two documents ``{ "encrypted<Type>": 0 }, { "encrypted<Type>": 6 }`` are returned.
 
 Case 6: encrypting a document greater than the maximum errors
 `````````````````````````````````````````````````````````````
 This test case should be skipped if the encrypted field is ``encryptedDoubleNoPrecision`` or ``encryptedDecimalNoPrecision``.
 
-Use ``clientEncryption.encrypt()`` to try to encrypt the value 201 with the matching ``RangeOpts`` listed in `Test Setup: RangeOpts`_ and these ``EncryptOpts``:
+Use ``clientEncryption.encrypt()`` to encrypt the value 201. Ensure the type matches that of the encrypted field.
+
+Encrypt using the following ``EncryptOpts``:
 
 .. code:: typescript
 
    class EncryptOpts {
-      keyId : <key1ID>
+      keyId : <key1ID>,
       algorithm: "RangePreview",
-      contentionFactor: 0
+      contentionFactor: 0,
+      rangeOpts: <RangeOpts for Type>,
    }
 
-Ensure 201 matches the type of the encrypted field. The error should be raised because 201 is greater than the maximum value in ``RangeOpts``.
+Assert that an error was raised because 201 is greater than the maximum value in ``RangeOpts``.
 
-Assert that an error was raised.
 
-Case 7: encrypting a document of a different type errors 
-````````````````````````````````````````````````````````
+Case 7: encrypting a value of a different type errors
+`````````````````````````````````````````````````````
 This test case should be skipped if the encrypted field is ``encryptedDoubleNoPrecision`` or ``encryptedDecimalNoPrecision``.
 
-For all the tests below use these ``EncryptOpts``:
+Use ``clientEncryption.encrypt()`` to encrypt the value 6 with a type that does not match that of the encrypted field.
+
+If the encrypted field is ``encryptedInt`` use a BSON double type. Otherwise, use a BSON int32 type.
+
+Encrypt using the following ``EncryptOpts``:
 
 .. code:: typescript
 
    class EncryptOpts {
-      keyId : <key1ID>
+      keyId : <key1ID>,
       algorithm: "RangePreview",
-      contentionFactor: 0
+      contentionFactor: 0,
+      rangeOpts: <RangeOpts for Type>,
    }
 
-If the encrypted field is ``encryptedInt`` encrypt ``{ "encryptedInt": { "$numberDouble": "6" } }``.
-Otherwise, encrypt ``{ "encrypted<Type>": { "$numberInt": "6" }``.
-Assert an error was raised.
+Ensure that ``RangeOpts`` corresponds to the type of the encrypted field (i.e. expected type) and not that of the value being passed to ``clientEncryption.encrypt()``.
+
+Assert that an error was raised.
+
 
+Case 8: setting precision errors if the type is not a floating point
```

Comment:
"floating point" seemed more accurate to describe a Decimal128 _or_ double type. I'm not sure if the original name was written before Decimal128 was added in a subsequent commit.